ocaml/idl/datamodel_host.ml

```diff
@@ -1079,10 +1079,23 @@ let uninstall_ca_certificate =
     ~doc:"Remove a TLS CA certificate from this host."
     ~params:
       [
-        (Ref _host, "host", "The host"); (String, "name", "The certificate name")
+        (Ref _host, "host", "The host")
+      ; (String, "name", "The certificate name")
+      ; ( Bool
```

Comment:
I think this would break existing clients, because the new param does not have a default. Use `~versioned_params` for that.